data/en/ukis_218.json

```
@@ -0,0 +1,4553 @@
+                                        {
+                                            "label": "Implementation of significant changes to marketing concepts or strategies",
+                                            "value": "Implementation of significant changes to marketing concepts or strategies",
+                                            "description": "For example a marketing concept or strategy that differs significantly from your enterprise’s existing marketing methods which has not been used before"
+                                        }
+                                    ]
+                                },
+                                {
+                                    "id": "answer3206-exclusive",
+                                    "mandatory": false,
+                                    "type": "Checkbox",
+                                    "label": "",
+                                    "description": "",
+                                    "options": [{
+                                        "label": "No significant changes",
+                                        "value": "No significant changes"
+                                    }]
+                                }
+                            ]
+                        }]
+                    }
+                ]
+            }]
+        },
+        {
+            "id": "section605",
+            "title": "Innovation Investment",
+            "groups": [{
+                "id": "group605",
+                "title": "Innovation Investment",
+                "blocks": [{
+                        "type": "Interstitial",
+                        "id": "innovation-investment-introduction",
+                        "title": "Innovation Investment",
+                        "description": "The next section covers <strong>investments in innovation</strong>.",
+                        "content": [{
+                            "description": "You will be asked to provide figures for expenditure on innovation activities in the calendar year 2018."
+                        }]
+                    },
+                    {
+                        "id": "block2428",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "question2428",
+                            "title": "Did {{ metadata.ru_name }} invest in <em>internal Research and Development </em>for the purposes of current or future innovation?",
+                            "description": "",
+                            "type": "General",
+                            "definitions": [{
+                                "title": "Internal Research and Development",
+                                "content": [{
+                                    "description": "Creative work undertaken within your business that increases knowledge for the developing new and improved goods or services and processes."
```

```suggestion
                                    "description": "Creative work undertaken within your business that increases knowledge for developing new and improved goods or services and processes."
```

Not in the slides but the grammar was wrong.

---

🤦‍♂️